src/medals/main.cljs

```diff
@@ -0,0 +1,39 @@
+(ns medals.main
+  (:require
+   [medals.constants :refer [MEDALCOLORS SORTVALUES]]
```

Comment:
We are not using `:refer`. Only `:as`. Please fix `medals.constants` to be

`[medals.constants :as constants]`

and use it as:

`constants/colors` 
`constants/sort-values` 

It also require renaming MEDALCOLORS SORTVALUES. I mean `color` and `sort-values`. It should be lowercased and we are using `-` as a separator. Name should be simple. Not so long because we are using unique namespaces, so we don't need to keep so drastically detailed naming convention.

**Not to forget**
And those packages should be sorted alphabetically.